x-pack/legacy/plugins/lens/public/xy_visualization_plugin/types.ts

```diff
@@ -177,10 +179,49 @@ export interface XYArgs {
 
 // Persisted parts of the state
 export interface XYState {
+  preferredSeriesType: SeriesType;
   legend: LegendConfig;
   layers: LayerConfig[];
   isHorizontal: boolean;
 }
 
 export type State = XYState;
 export type PersistableState = XYState;
+
+export const visualizationTypes: VisualizationType[] = [
+  {
+    id: 'bar',
+    icon: 'visBarVertical',
+    label: i18n.translate('xpack.lens.xyVisualization.verticalBarLabel', {
+      defaultMessage: 'Vertical Bar',
+    }),
+  },
+  {
+    id: 'bar_stacked',
+    icon: 'visBarVertical',
+    label: i18n.translate('xpack.lens.xyVisualization.stackedVerticalBarLabel', {
+      defaultMessage: 'Stacked Vertical Bar',
```

Comment:
It's really weird that we can see a label saying "Vertical Bar" but then check the "Rotate 90º" button and it's a horizontal vertical bar. I would either remove the "Vertical" labeling or go back to having a "Horizontal bar" be a separate chart type

Reply:
Changed that to only have "Bar" in the popover as having two buttons for each chart type would get too busy. The orientation can be changed in the chart settings